lib/shared/src/lexicalEditor/editorState.test.ts

```diff
@@ -162,6 +170,181 @@ describe('editorStateFromPromptString', () => {
+                                {
+                                    type: 'contextItemMention',
+                                    version: 1,
+                                    contextItem: {
+                                        type: 'file',
+                                        uri: 'file:///foo/bar/baz.go',
+                                        content: undefined,
+                                        source: ContextItemSource.User,
+                                        range: {
+                                            start: { line: 2, character: 0 },
+                                            end: { line: 5, character: 0 },
+                                        },
+                                    },
+                                    isFromInitialContext: false,
+                                    text: 'baz.go:3-5',
+                                },
+                                {
+                                    detail: 0,
+                                    format: 0,
+                                    mode: 'normal',
+                                    style: '',
+                                    text: ' do?',
+                                    type: 'text',
+                                    version: 1,
+                                },
+                            ],
+                            direction: null,
+                            format: '',
+                            indent: 0,
+                            type: 'paragraph',
+                            version: 1,
+                        },
+                    ],
+                    direction: null,
+                    format: '',
+                    indent: 0,
+                    type: 'root',
+                    version: 1,
+                },
+            }),
+        }
+        expect(editorStateToText(mockEditorState as any)).toBe('What does foo/bar/baz.go:3-5 do?')
+    })
+
+    test('converts editor state with multiple file context items', () => {
+        const mockEditorState = {
+            toJSON: () => ({
+                root: {
+                    children: [
+                        {
+                            children: [
+                                {
+                                    detail: 0,
+                                    format: 0,
+                                    mode: 'normal',
+                                    style: '',
+                                    text: 'Compare ',
+                                    type: 'text',
+                                    version: 1,
+                                },
+                                {
+                                    type: 'contextItemMention',
+                                    version: 1,
+                                    contextItem: {
+                                        type: 'file',
+                                        uri: 'file:///foo/bar/file1.go',
```

Comment:
```suggestion
                                        uri: testFileUri('/foo/bar/file1.go'),
```
i think this should fix the test on windows